### ISSUE TITLE
Question: node_modules folder in other place

### ISSUE THREAD

**Issue by Piioo** [FormidableLabs/enzyme-matchers#244]
I use the .yarnrc file to specify where the node_modules folder is located. I don't have to know where the place is, yarn knows.

How do I have to specify the **setupTestFrameworkScriptFile** in the package.json?

**Comment by pascalduez** [comment]
Hi,

`setupTestFrameworkScriptFile: 'jest-enzyme'` is enough.

**Comment by Piioo** [comment]
Yes this is working fine, when the node_modules folder is in the project **\<rootDir\>/node_modules/....**, but when it is placed somewhere else I got:

````
yarn run v1.7.0
$ jest
● Validation Error:

  Module jest-enzyme in the setupTestFrameworkScriptFile option was not found.
````
